main.tf

```
@@ -134,7 +134,7 @@ resource "aws_launch_configuration" "this" {
 
 module "bitbucketdb" {
   source  = "rhythmictech/rds-postgres/aws"
-  version = "~> 3.0.1"
+  version = "4.1.0-rc1"
```

```suggestion
  version = "~> 4.1.0"
```
do we want to pin to the RC?